### PR TITLE
autopep8 auto formatting

### PR DESCRIPTION
**Pull request by fschlueter**
formatting

### CODE REVIEW

File: NuRadioReco/modules/io/coreas/readCoREASShower.py [hunk 1]
```diff
@@ -103,20 +108,26 @@ def run(self):
...
                 station.set_sim_station(sim_station)
                 evt.set_station(station)
                 if self.__det is not None:
                     position = observer.attrs['position']
                     antenna_position = np.zeros(3)
-                    antenna_position[0], antenna_position[1], antenna_position[2] = -position[1] * units.cm, position[0] * units.cm, position[2] * units.cm
-                    antenna_position = cs.transform_from_magnetic_to_geographic(antenna_position)
+                    antenna_position[0], antenna_position[1], antenna_position[2] = - \
+                        position[1] * units.cm, position[0] * \
+                        units.cm, position[2] * units.cm
```
Comment (sjoerd-bouma) [inline NuRadioReco/modules/io/coreas/readCoREASShower.py:128]: [No way this is PEP8](https://peps.python.org/pep-0008/#maximum-line-length)! I think the PEP8 way would be to either turn this into a tuple
```
... = (
    -position[1] * units.cm,
    position[0] * units.cm,
    position[2] * units.cm
)
```
or, alternatively, just turn this into three separate lines. `\` is ugly!

**Review by sjoerd-bouma — CHANGES_REQUESTED** [review]

File: NuRadioReco/modules/io/coreas/readCoREASShower.py (hunk 1 above)
Comment (fschlueter) [reply NuRadioReco/modules/io/coreas/readCoREASShower.py]: better?

Comment (sjoerd-bouma) [reply NuRadioReco/modules/io/coreas/readCoREASShower.py]: Much <3

**Review by sjoerd-bouma — APPROVED** [review]
Thanks Felix!

File: NuRadioReco/modules/io/coreas/readCoREASShower.py (hunk 1 above)
Comment (fschlueter) [reply NuRadioReco/modules/io/coreas/readCoREASShower.py]: I wonder why my formatter did not complain. I agree that was not good style at all.

**Pull request merged**